johnzon-mapper/src/main/java/org/apache/johnzon/mapper/MapperConfig.java

```diff
@@ -98,6 +98,8 @@ public Object fromJson(JsonValue jsonObject, Type targetType, MappingParser pars
 
     private final SnippetFactory snippet;
 
+    private final Function<MapperConfig, Mappings> mappingsFactory;
+
     //CHECKSTYLE:OFF
     @Deprecated
     public MapperConfig(final LazyConverterMap adapters,
```

Comment:
Is it okay to touch the other constructor of MapperConfig? (the one not used in ExtendMappingTest)
Or do I add a third one and mark the current non-deprecated one as deprecated?

Reply:
For the scope I have in mind it would be ok but it can be a limited scope (understand I will not fight if you modify the other but I have no real idea it can break other people).